MachineTools/MRMachineTool.cpp

```diff
@@ -1,3 +1,10 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * Copyright 2020-2021 (c) Christian von Arnim, ISW University of Stuttgart (for umati and VDW e.V.)
+ */
```

Comment:
I authored the original version of this file in 2020.
Suggestion: add resp. Copyright note.

Reply:
Overlooked, fixed!